src/gltf-component-mappings.js

```diff
@@ -24,6 +24,8 @@ function registerRootSceneComponent(componentName) {
 
 registerRootSceneComponent("fog");
 
+AFRAME.GLTFModelPlus.registerComponent("troika-text", "troika-text");
```

Comment:
We need to either rename the component in Spoke or map this to the updated "text" component so the Spoke Troika PR works. I've already done that in a branch with a few other updates on that PR so I can push it similarly to what you have done with this one.

Reply:
I have renamed the component in the new Spoke PR so we can probably remove this mapping.

Reply:
Ah yeah that's actually a bug, no component called `troika-text` exists anymore with this PR, so this shouldn't be there. I decided it was worth just replacing wholesale since its not exposed in Spoke and I very highly doubt it was being used in many Blender scenes since it was much too hard to configure correctly.